books/refgram/src/from_scratch/intro.md

```diff
@@ -1,25 +1,22 @@
 # Eberban from scratch
 
-One of the long term goal of Eberban is to have its vocabulary entirely defined
-in Eberban itself, and not realy on external or _opaque_ predicates.
+One of the long term goals of Eberban is to have its vocabulary entirely defined
+in Eberban itself, and to not rely on external or _opaque_ predicates.
 
-To achieve such goal Eberban is designed following the following principles:
+To achieve such a goal, Eberban is designed to the following principles:
 
 - A subset of the complete Eberban grammar is called the __core grammar__, and
   provides only a basic logic framework on which can be built the rest of the
   language.
-- Grammar out of the __core grammar__ is then defined as _syntaxic sugar_ over
-  some predicates that must be defined only using the __core grammar__. Some
-  parts of the complete grammar definitions can use non-core grammar only if
-  there are no cyclic dependencies.
-- Once the complete grammar is defined, more vocabulary can be added to
-  model required for speech and speaking about wanted subjects.
+- Grammar defined by core grammar predicates is known as __syntactic sugar__.
```

Comment:
I don't think we can say that it is defined **by** the core grammar. We define predicates using the core grammar, and non-core grammar is syntactic sugar over those defined predicates. Also note that "syntactic sugar" is not the name of non-core grammar, it is just a statement of what they are:
https://en.wikipedia.org/wiki/Syntactic_sugar

Reply:
```suggestion
```
Please remove this sentence as we discussed before. The rest looks good.